procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,88 @@
+# Council Synchronous Meetings
+
+This documents procedures, norms, and logistics of the Council's sync meetings.
+
+## Frequency
+
+The Council meets once a week.
+
+After the 14th of July, 2023, the Council will re-evaluate whether this frequency still makes sense or whether it should be changed.
+
+## Location
+
+The meeting happens in some video call software (e.g., Google Meet).
+
+## Quorum
+
+The meeting has reached quorum if at *most* 2 Council representatives (and their alternates) are not present (i.e., N - 2).
+
+If quorum is not reached, then the sync meeting is canceled and the agenda is gone through asynchronously in chat.
+
+## Minutes
+
+Detailed minutes must be taken of the meeting even if the meeting was otherwise recorded. The meeting notes must be made public as soon as possible (preferably latest one week after the meeting), but before doing so Council members are asked to review those meeting notes to ensure that the notes reflect their understanding of what occurred in the meeting. Once consensus is reached that the meeting notes are accurate, the notes are released for public viewing. 
+
+**TODO**: decide on location for meeting notes
+
+## Recording
+
+All meetings are recorded, but consensus must be reached after the meeting on what do with the recording. The Council has no obligation to do anything with the recording. If after one week, consensus on what to do with the recording has not been reached, the default is to delete the recording. 
+
+Recordings are usually used to ensure that the notes are correct and to optionally create automatic transcripts. Recordings can also *optionally* be made public, but only with consensus of all of those present on the recording.
+
+## Meeting roles
+
+The meeting has the following assigned roles:
+
+- **Facilitator**: the facilitator is in charge of pushing discussion along, calling on hands, formally asking the group whether there are any objections to a particular motion, and keeping the discussion on agenda.
+- **Scribe(s)**: the scribes are in charge of taking notes on what was discussed in the meeting including noting any action items or follow up questions.
```

Comment:
This may be better as a follow-up policy document, but I liked @Manishearth's description of the scribing process they've used in W3C meetings (I've lost the Zulip link for where this was discussed). The idea was to have one person in charge of scribing, other's contribute, and when the scribe is called on to speak there's an explicit delegation to an alternate scribe while the main scribe speaks.

Reply:
This sounds great. I'd like for us to try this out and adopt the process before documenting it as the process though. I'll put this in the agenda for the next meeting to mention that we might want to try something like this.